packages/components/inputs/rich-text-input/src/rich-text-input.story.js

```diff
@@ -14,7 +14,7 @@ import Readme from '../README.md';
 
 // Create our initial value...
 
-const initialValue = '<h1>H1 heading</h1>';
+const initialValue = '<h1>H1 <u>heading</u></h1>';
```

Comment:
I changed this as suggested by @emmenko to enable proper testing of mark button toggling state upon applying initial focus